client/mysqlbinlog.cc

```diff
@@ -1684,7 +1685,7 @@ static struct my_option my_options[] =
    "Stop reading the binlog at position N. Applies to the last binlog "
    "passed on the command line.",
    &stop_position, &stop_position, 0, GET_ULL,
-   REQUIRED_ARG, (longlong)(~(my_off_t)0), BIN_LOG_HEADER_SIZE,
+   REQUIRED_ARG, stop_position_default, BIN_LOG_HEADER_SIZE,
```

Comment:
Let's make the `max_value` arg also use `stop_position_default` so it is clear from the go that the default is the max